pyproject.toml

```
@@ -73,6 +73,7 @@ dependencies = [
     "zstandard==0.18.0",
 ]
 dynamic = ["version"]
+license-files = ["LICENSE*"]
```

```suggestion
license-files = ["LICENSE"]
```
maybe this would work? Or `"LICEN[CS]E*"`? Or maybe order matters?